@xen-orchestra/web/src/pages/backup/[id]/configuration.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="backup-jobs-configuration">
+    <BackupJobsGeneralInformation :backup-job />
+    <BackupJobsSettings :backup-job />
+    <BackupJobsSchedules :backup-jobs="backupJobsSchedules" />
```

Comment:
Sorry to request a rollback, but I think `<BackupJob*` (without the "s") for the components' names better reflect their purpose. We want to display the information, settings and schedules for only one job.
```suggestion
    <BackupJobGeneralInformation :backup-job />
    <BackupJobSettings :backup-job />
    <BackupJobSchedules :backup-job-schedules />
```
I also changed the prop's name, see my other comment in the script part.